.github/workflows/ci.yml

```diff
@@ -66,6 +66,26 @@ jobs:
           cache-from: type=registry,ref=${{ env.DOCKER_IMAGE_TAG }}
           cache-to: type=inline
 
+  deploy:
+    if: startsWith(github.ref, 'refs/tags/v')
+    needs: [ prettier, es-lint, tests ]
+    runs-on: ubuntu-latest
+    env:
+      DOCKERHUB_ORG: safeglobal
+      DOCKERHUB_PROJECT: safe-gelato-relay-service
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@v3
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Deploy tag
+        run: bash scripts/deploy_docker.sh ${GITHUB_REF##*/}
+
   autodeploy:
     runs-on: ubuntu-latest
     needs: [ docker-publish ]
```

Comment:
I think this ref should be changed to `docker-publish-staging` now 🙂

Reply:
Oh, nice catch!